person.rb

```diff
@@ -29,4 +33,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Same here, Please edit this method to make it create a new rental and the Person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.